packages/styles/global/variables.scss

```diff
@@ -1,18 +1,21 @@
 :root {
-  --space-xxs: 4px;
-  --space-xs: 8px;
-  --space-sm: 16px;
-  --space: 24px;
-  --space-md: 32px;
-  --space-lg: 48px;
-  --space-xlg: 64px;
-  --screen-xs: 360px;
-  --screen-sm: 768px;
-  --screen-md: 992px;
-  --screen-lg: 1280px;
-  --screen-xlg: 1440px;
-  --screen-max-container: 1140px;
+  /* Spacings
+  * ------------------------- */
+  --spacing-xxsmall: 4px;
+  --spacing-xsmall: 8px;
+  --spacing-small: 12px;
+  --spacing-base: 16px;
+  --spacing-medium: 20px;
+  --spacing-large: 24px;
+  --spacing-xlarge: 29px;
```

Comment:
question: this is correct? shouldn't it be 28px?

Reply:
THANKS! hahahaha 😱